src/ontology/mondo-ingest.Makefile

```diff
@@ -362,16 +356,20 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
-tmp/mondo.owl tmp/mondo.sssom.tsv:
+tmp/mondo_repo_built:
```

Comment:
I created a separate goal for checking out the MONDO repo as the `make` pipeline did have some issues with the "multi-target" solution before (essentially rerunning the goal twice).